cloud-storage/xml-api/serviceaccount-appengine-sample/src/main/java/com/google/api/client/sample/storage/appengine/serviceaccount/StorageSample.java

```diff
@@ -0,0 +1,90 @@
+//[START all]
+/*
+ * Copyright (c) 2012 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.api.client.sample.storage.appengine.serviceaccount;
+
+import com.google.api.client.googleapis.extensions.appengine.auth.oauth2.AppIdentityCredential;
+import com.google.api.client.http.GenericUrl;
+import com.google.api.client.http.HttpRequest;
+import com.google.api.client.http.HttpRequestFactory;
+import com.google.api.client.http.HttpResponse;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.http.javanet.NetHttpTransport;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.util.Arrays;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Google Cloud Storage Service Account App Engine sample.
+ *
+ * @author Marc Cohen
+ */
+public class StorageSample extends HttpServlet {
+
+  private static final long serialVersionUID = 1L;
+
+  private static final String GCS_URI = "http://commondatastorage.googleapis.com";
+
+  /** Global configuration of Google Cloud Storage OAuth 2.0 scope. */
+  private static final String STORAGE_SCOPE =
+      "https://www.googleapis.com/auth/devstorage.read_write";
+
+  /** Global instance of the HTTP transport. */
+  private static final HttpTransport HTTP_TRANSPORT = new NetHttpTransport();
+
+  /** Global instance of HTML reference to XSL style sheet. */
+  String XSL = "\n<?xml-stylesheet href=\"/xsl/listing.xsl\" type=\"text/xsl\"?>\n";
+
+  @Override
+  protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
+
+    try {
+      AppIdentityCredential credential = new AppIdentityCredential(Arrays.asList(STORAGE_SCOPE));
```

Comment:
You need a newer version of com.google.api-client. Maybe you can try [1.20.0](http://search.maven.org/#artifactdetails%7Ccom.google.api-client%7Cgoogle-api-client%7C1.20.0%7Cjar).